backend/src/routes/__tests__/sessions.test.js

```diff
@@ -70,17 +65,58 @@ afterAll((done) => {
   });
 });
 
-describe('GET /sessions/:id', () => {
-  it('can not find a session without a id', async (done) => {
-    let err;
-    try {
-      await axios.get(`http://localhost:${port}/sessions`);
-    } catch (error) {
-      err = error;
-    }
-    expect(err).toBeDefined();
-    done();
-  });
-});
+describe('PATCH', () => {
+  it('should be able to update the session object', async () => {
+    const body = {
+      preferences: {},
+      results: [],
+    };
+
+    let response = await axios.post(`http://localhost:${port}/sessions`, body);
+    let returnTask = response.data;
+
+    const updateBody = {
+      isFinished: true,
+      preferences: {
+        location: 'Sydney',
+        distance: 10,
+        cuisines: ['Thai', 'Japanese', 'Chinese'],
+        price: 20,
+        timer: 600,
+        coordinates: {
+          lat: 34.6424325,
+          lng: 10.2343462,
+        },
+      },
+
+      results: [
+        {
+          name: 'Thai Restaurant',
+          location: '1 Thai Street',
+          numberOfVotes: 5,
+        },
+        {
+          name: 'Japanese Restaurant',
+          location: '1 Japan Street',
+          numberOfVotes: 3,
+        },
+        {
+          name: 'Chinese Restaurant',
+          location: '1 China Street',
+          numberOfVotes: 7,
+        },
+      ],
+    };
 
+    response = await axios.patch(
+        `http://localhost:${port}/sessions/${returnTask.truncCode}`,
+        updateBody,
+    );
 
+    returnTask = response.data;
+
+    expect(returnTask.isFinished).toBe(true);
+    expect(returnTask.preferences.location).toBe('Sydney');
+    expect(returnTask.results[1].name).toBe('Japanese Restaurant');
```

Comment:
Nick work! It would be good to add a expect for status code.
`expect(response.status).toBe(200);`